examples_tests/47.DerivMapTest/main.cpp

```diff
@@ -228,9 +228,12 @@ static core::smart_refctd_ptr<asset::ICPUImage> createDerivMapFromHeightMap(asse
 	state.axisWraps[1] = _vwrap;
 	state.axisWraps[2] = asset::ISampler::ETC_CLAMP_TO_EDGE;
 	state.borderColor = _borderColor;
+	state.enableLUTUsage = true;
```

Comment:
shouldn't even be an option!

Reply:
I forgot to remove this, as it is now, ex47 will not compile.

Reply:
Actually, I have already removed this, as seen in the latest commit [here](https://github.com/Devsh-Graphics-Programming/Nabla/pull/343/commits/232a5a12ebdf85a5c5ef73fcfa89b5add558b97d). Not sure why it is showing that.